src/messages/FileMessage.vue

```diff
@@ -25,12 +64,39 @@ export default {
     messageColors: {
       type: Object,
       required: true
+    },
+  },
+  methods:{
+    humanReadableFileSize(aSize) {
+      const prettyBytes = require('pretty-bytes');
+      return prettyBytes(aSize);
+    },
+    truncateFilename(filename, maxLength) {
+      if(filename.length > maxLength) {
+        const smartTruncate = require('smart-truncate');
+        return smartTruncate(string, maxLength, {position: 10});
+      } else {
+        return filename
+      }
     }
+  },
+  components:{
+    ApplicationWord,
+    ApplicationPdf,
+    ApplicationPng,
+    ApplicationExcel,
+    ApplicationPowerPoint,
+    ApplicationZip,
+    ApplicationDefault,
+    ApplicationJpg,
+    ApplicationSvg,
+    ApplicationCsv,
+    DownloadIcon
   }
 }
 </script>
 
-<style scoped>
+<style lang="scss">
```

Comment:
Let's keep `scoped` attr. `/deep/` selectors can be used where it'd be an issue.